routers/repo/pull.go

```diff
@@ -516,6 +516,18 @@ func MergePullRequest(ctx *context.Context) {
 
 	pr.Issue = issue
 	pr.Issue.Repo = ctx.Repo.Repository
+
+	noDeps, err := models.IssueNoDependenciesLeft(issue)
+	if err != nil {
+		return
+	}
+
+	if !noDeps {
+		ctx.Flash.Error("You need to close all issues blocking this pull request before you can merge it!")
```

Comment:
Flashed text should be a locale for i18n (also elsewhere).

Reply:
Done.